Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -2032,3 +2034,53 @@ void MainWindow::set_facegraph_mode_adapter(bool is_polyhedron)
     set_face_graph_default_type(SURFACE_MESH);
 }
 
+void MainWindow::exportStatistics()
+{
+  std::vector<Scene_item*> items;
+  Q_FOREACH(int id, getSelectedSceneItemIndices())
+  {
+    Scene_item* s_item = scene->item(id);
+    items.push_back(s_item);
+  }
+
+  QString str;
+  Q_FOREACH(Scene_item* sit, items)
+  {
+    CGAL::Three::Scene_item::Header_data data = sit->header();
+    if(data.titles.size()>0)
+    {
+      int titles_limit =0;
+      int title = 0;
+      str.append(QString("%1: \n").arg(sit->name()));
+      for(int j=0; j<data.categories.size(); j++)
+      {
+        str.append(QString("  %1: \n")
+                   .arg(data.categories[j].first));
+        titles_limit+=data.categories[j].second;
+        for(;title<titles_limit; ++title)
+        {
+          str.append(QString("    %1: ").arg(data.titles.at(title)));
+          str.append(QString("%1\n").arg(sit->computeStats(title)));
+          title++;
+        }
```

Comment:
@maxGimeno: There is a warning about this loop in recent testsuites:
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-59/Polyhedron_Demo/TestReport_lrineau_ArchLinux-clang-CXX14.gz
```
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-clang-CXX14/test/Polyhedron_Demo/MainWindow.cpp:2066:11: warning: variable 'title' is incremented both in the loop header and in the loop body [-Wfor-loop-analysis]
          title++;
          ^
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-clang-CXX14/test/Polyhedron_Demo/MainWindow.cpp:2062:36: note: incremented here
        for(;title<titles_limit; ++title)
                                   ^
```

Is that wanted? Anyway, even if that is wanted, I agree with the warning that it is confusing and it should be replaced by a single `title+=2`.

Reply:
It was absolutely not wanted. Fixed in #2532.